<launch>
  <node pkg="rviz" type="rviz" name="rviz"
    args="-d $(find ar_pose)/launch/live_multi.rviz"/>
  <node pkg="tf" type="static_transform_publisher" name="world_to_cam" 
    args="0 0 0.5 -1.57 0 -1.57 world camera 1" />
  <node ns="camera" pkg="image_proc" type="image_proc" name="image_proc"/>
  <node ns="camera" pkg="uvc_camera" type="camera_node" name="uvc_camera"
    output="screen">
    <param name="width" type="int" value="320" />
    <param name="height" type="int" value="240" />
    <param name="fps" type="int" value="30" />
    <param name="frame" type="string" value="camera" />
    <param name="device" type="string" value="/dev/video1" />
    <param name="camera_info_url" type="string"
      value="file://$(find uvc_camera)/camera_calibration.yaml" />
  </node>
  <node name="ar_pose" pkg="ar_pose" type="ar_multi" respawn="false"
    output="screen">
    <param name="marker_pattern_list" type="string"
      value="$(find ar_pose)/data/object_4x4"/>
    <param name="threshold" type="int" value="100"/>
  </node>
</launch>
